src/components/navbar/Unconnected/Logo.tsx

```diff
@@ -0,0 +1,16 @@
+import { displayFromSm, displayFromXsToSm } from 'helpers/visibilityClassnames'
+import SealWallet from 'icons/SealWallet'
+import SmallSeal from 'icons/SmallSeal'
+
+export default function () {
+  return (
+    <>
+      <span className={displayFromXsToSm}>
```

Comment:
This is OK, I think

Reply:
Btw, why don't we pass `connected` as prop so we don't need two separate components for this?

Reply:
Changed it